src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,62 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+// get roomid(DisolayIframe(rid)) and put rid id in iframe link
+// <iframe style="height:80%" src="localhost:3000/livechat/roomId"></iframe>
+// For example, for now I have add link of jitsi and its working fine, similarly we can add link of our video/audio call implemented with webrtc.
```

Comment:
Please make sure to remove these comments in your final PR. We can't keep this PR in production